package.json

```diff
@@ -1970,12 +1970,9 @@
                 },
                 "jupyter.pylanceHandlesNotebooks": {
                     "type": "boolean",
-                    "default": false,
+                    "default": true,
                     "description": "Determines if pylance's experimental notebook support is used or not.",
```

Comment:
should we change this description?
ie is this feature still experimental ?

Reply:
Thanks. I didn't notice the word experimental in the description. @heejaechang, is the new string an accurate description of the setting?

Reply:
Can/should the setting be hidden so it doesn't show up in the settings UI?